pubsub/topic.go

```diff
@@ -474,6 +474,14 @@ func (t *Topic) Stop() {
 	t.scheduler.FlushAndStop()
 }
 
+// Flush ensures all remaining messages are sent.
+func (t *Topic) Flush() {
```

Comment:
This could also take a bool to stop perhaps? Then just recommend calling t.Flush(true) instead of t.Stop(). This is fine though as there is already a means to stop and flush.

Reply:
I prefer to leave the methods separate as it's a bit more explicit and potentially less confusing. However, if there's precedent to take a bool here in other functions, I'd be happy to add it in.